src/turbomind/models/llama/LlamaBatch.h

```diff
@@ -220,6 +220,7 @@ class LlamaBatch {
     const int      tp_rank_;
     const DataType data_type_;
     const bool     debug_;
+    const bool     is_driver_;
```

Comment:
Consider renaming `is_driver_` to be more specific. The current name is vague - what exactly is it driving or controlling?